src/generator/operations.ts

```diff
@@ -203,6 +203,36 @@ function generateMultiRespComment(op: Operation): string {
   return `// Possible return types are ${returnTypes.join(', ')}\n`;
 }
 
+function getZeroReturnValue(op: Operation, apiType: 'api' | 'op'): string {
+  if (!op.responses) {
+    // no responses return *http.Response
+    return 'nil';
+  }
+  let returnType = 'nil';
+  if (isLROOperation(op)) {
+    if (apiType === 'op') {
+      // the operation returns an *azcore.Response
+      returnType = 'nil';
```

Comment:
Just curious, when does this case happen?

Reply:
For operations that don't return a model (the `httpinfrastructuregroup` has a lot of examples).

Reply:
Ok I didnt see one with an LRO, but seems like a good idea just in case 👍